### DISCUSSION TITLE
Some of my text is missing from the matrix?

### DISCUSSION THREAD

**Discussion by dufus87** [ccbogel/QualCoder#908] [Q&A] [answered]
Hi i have been trying to create a matrix with my data but it seems the export doesn't include all my categories / themes? Even if the frequency report says there's at least 1 for a certain theme or category, the matrix for that certain section is empty. What do I do?

**Comment by ccbogel** [comment]
Hi. I'm presuming you are using the latest 3.5 release, and that it concerns text coding only. 
It's hard to answer without seeing the data.
Firstly, can you identify the specific text coding that is showing in the frequency report and not showing in the Matrix report?
Secondly is the Matrix Report being prepared by using 'all coders' and not a specific coder?
Let me know. then if still an issue, I might have to view the project to see if I can track down the issue.

**Comment by dufus87** [comment]
Hi! Thanks so much for reaching out.

Yes, I am using 3.5 and it is regarding text coding.
I have attached the image it's Working with AI: New team compositions...
As for the matrix report, I am the only coder

The image only shows 1 code currently missing, from scrolling there are 20
that show up in the frequency but dont show up in the matrix.
I am also interested in examining my data using the cases function but I am
unsure if this same issue that i see when I use codes by case (missing
data) also affects using categories by case -- definitely there are some
empty spaces but I have yet to check if they're empty because it is not
relevant or because it is missing data as well.

**Reply by ccbogel** [reply]
OK, where is the image ? and can you show the code tree, as maybe the code is not within a category ? perhaps

**Reply by dufus87** [reply]
<img width="1434" alt="Screenshot 2024-05-10 at 07 31 18" src="https://github.com/ccbogel/QualCoder/assets/163316418/3d42aed3-3532-4272-8263-9c6428c1b0a2">
<img width="654" alt="Screenshot 2024-05-10 at 07 30 46" src="https://github.com/ccbogel/QualCoder/assets/163316418/080cccd9-d1f0-4a88-acb0-29fc3cca0edc">

**Reply by ccbogel** [reply]
ok and the Ways o fworking is missing a code? Is there only 3 files (3 rows)

**Reply by dufus87** [reply]
The working with AI: New Team Compositions quote / code is not showing up on the matrix
There are 45 files (word documents), there are 3 rows because I sorted them by case which are the years

**Reply by ccbogel** [reply]
Ahhhh.... is that file that the quote is in - a part of a case? ( a part of the 3 cases).

**Reply by dufus87** [reply]
Yes all the files are assigned cases each year has 15 files , and there are a total of 45 files

**Reply by dufus87** [reply]
<img width="411" alt="Screenshot 2024-05-10 at 07 52 32" src="https://github.com/ccbogel/QualCoder/assets/163316418/c6cd68f3-acf5-43a9-a952-77cbefd3bfe2">

**Reply by ccbogel** [reply]
hmm. its hard to see what the issue is right now. but to check the coded bit shows up in the files by codes, and / or the files by category ? (at least for when that specific file is selected, dont do case selection for this test).

**Reply by dufus87** [reply]
Still didnt show up 
<img width="603" alt="Screenshot 2024-05-10 at 08 07 32" src="https://github.com/ccbogel/QualCoder/assets/163316418/bb67a229-340e-4f5b-b658-4e936d78af41">
<img width="1470" alt="Screenshot 2024-05-10 at 08 08 11" src="https://github.com/ccbogel/QualCoder/assets/163316418/fd23cf7a-7e60-4e9d-8705-bd560491b94a">

**Reply by ccbogel** [reply]
hmm. Without the Matrix, Does that coded bit show up when only selecting for that code across the files (in the second pane)

**Reply by dufus87** [reply]
Yes 
<img width="1168" alt="Screenshot 2024-05-10 at 08 16 42" src="https://github.com/ccbogel/QualCoder/assets/163316418/435dfa70-bc59-493e-a01b-033820ad1bbf">

**Reply by ccbogel** [reply]
Im having a quick look at the QualCoder code to see if anything leaps out. One thing which might seem quirky - is that the only code name with the ampersand sign in it? as this might be affecting display

**Reply by dufus87** [reply]
It's not the only one and still there are others without the ampersand that dont show the quotes / codes 
<img width="540" alt="Screenshot 2024-05-10 at 08 26 39" src="https://github.com/ccbogel/QualCoder/assets/163316418/f7859b9c-af52-4ca5-ad58-63eed7120a4a">
<img width="291" alt="Screenshot 2024-05-10 at 08 26 50" src="https://github.com/ccbogel/QualCoder/assets/163316418/58b702e0-82a5-48a1-8494-409749121c6f">

**Reply by ccbogel** [reply]
OK. Well that is not the issue then.
This is tricky to identify and fix without testing directly with the issue within your project data.

I guess the only practical (but not ideal) way would be to export the matrix to excel as you intended to do, and back fill the missing quote(s) from the second pane.

(p.s. I have to go for now, but will check in later/or over weekend)

**Reply by dufus87** [reply]
Not sure if this is helpful but I looked at the frequencies again, for the code "Working with AI (general" , it should appear 6 times. When I run the coding report with ONLY THE FILES AND CODE (code by files) selected BOTH the Matrix and The text part (second pane) show all 6 so its correct. 

However, if I add cases (whether year, type of company, or region) only 3 of the quotes show up in BOTH the text part and the matrix.

Thanks for checking in, I really appreciate the time you took to respond, I guess it is what must be done. (backfilling)

**Reply by ccbogel** [reply]
OK thanks for checking, theres some issue there I have to try and fix

**Comment by dufus87** [comment]
Thank you!

**Comment by cyanbae** [comment]
Hi, I had the same problem and what solved my issue was the length of the code names.
Code names have to be somehow short enough to be shown on the matrix. Hope this could be fixed in the next update!
If the code name is too long it gets shortened with "..." on the code list and I think that confuses the search when you create a matrix.

**Comment by ccbogel** [comment, marked as answer]
Yes Long codes names were truncated, causing a problem. I have fixed this now in the latest code for QC 3.7.
I will add a warning in the wiki about it.
